experiment/simulation/css/main.css

```diff
@@ -1 +1,55 @@
-/* You CSS goes in here */
\ No newline at end of file
+.page {
+	overflow-x: scroll;
+}
+
+.v-canvas-container {
+	width: 100%;
+	height: 100%;
+}
+
+.v-datalist-container {
+	height: 100%;
+}
+
+.v-slider-full {
+	width: 80%;
+}
+
+.input-form {
+	display: none;
+	color: black;
+}
+
+.add {
+	font-size: 0.90rem; 
+}
+
+.apparatus {
+	order: 1;
+}
+
+.simulation {
+	order: 2;
+}
+
+.observations {
+	order: 3;
+}
+
+@media screen and (max-width: 1023px) {
+	.workspace {
+		flex-wrap: wrap;
+	}
+
+	.apparatus {
+		order: 1;
```

Comment:
It's interesting that Bulma doesn't offer an order class whereas bootstrap does.

Reply:
Ya, I searched but could not find anything in Bulma for reordering.